src/libxively/xi_context.c

```diff
@@ -0,0 +1,249 @@
+/* Copyright (c) 2003-2017, LogMeIn, Inc. All rights reserved.
+ *
+ * This is part of the Xively C Client library,
+ * it is licensed under the BSD 3-Clause license.
+ */
+
```

Comment:
is this just things that moved out of xively.c ?

Reply:
or should I look at it more closely?

Reply:
yes, moved out of xively.c and minor refactor. Just peeled the xi_context_handle_t to the very end since the internal user (edge device contet) does not need the memory address <--> handle mapping.

Reply:
👌